src/content/docs/cheerpj2/index.md

```diff
@@ -1,25 +1,39 @@
 ---
 title: CheerpJ 2
-subtitle: Convert Java to WebAssembly and JavaScript
+subtitle: JVM replacement for the browser
 ---
 
-CheerpJ is a Java bytecode to WebAssembly and JavaScript compiler, compatible with 100% of Java, which allows to compile any Java SE application, library or Java applet into a WebAssembly/JavaScript application.
+CheerpJ is a **drop-in replacement** for the JVM, and is compatible with 100% of Java 8, including Swing, reflection and dynamic class loading.
 
-Please visit the project [website](https://leaningtech.com/cheerpj/).
+![](/cheerpj2/assets/cheerpj_visual_2.png)
 
-**Download latest version**: [![Latest version](https://img.shields.io/badge/cheerpj-2.3-green.svg)](https://leaningtech.com/download-cheerpj/) [![Latest version changelog](https://img.shields.io/badge/2.3-changelog-green.svg)](/cheerpj2/changelog)
+With CheerpJ, you can:
 
-**Link to latest runtime**
+- Run existing **Java applications** in the browser with no changes
+- Include **Java applets** in webpages without legacy plugins
+- Migrate **Java Web Start / JNLP** applications to work on modern systems
+- Use Java libraries in JavaScript/TypeScript seamlessly
 
-```
-https://cjrtnc.leaningtech.com/2.3/loader.js
-```
+## Getting started
 
-If you are unsure how to start, try our [tutorial](/cheerpj2/getting-started/Tutorial).
+Know what you're building? Jump straight to the relevant tutorial:
 
-## What is CheerpJ?
+<div class="not-prose grid grid-cols-2 font-medium gap-2 text-stone-100">
+  <a href="/cheerpj2/getting-started/Java-app" class="px-8 py-6 bg-stone-800 hover:bg-stone-700 text-lg">
+    Application
+  </a>
+  <a href="/cheerpj2/getting-started/Java-applet" class="px-8 py-6 bg-stone-800 hover:bg-stone-700 text-lg">
+    Applet
+  </a>
+  <!--<a href="/cheerpj2/getting-started/Java-app" class="px-8 py-6 bg-stone-800 hover:bg-stone-700 text-lg">
+    Java Web Start / JNLP
+  </a>-->
+  <a href="/cheerpj2/getting-started/Java-library" class="px-8 py-6 bg-stone-800 hover:bg-stone-700 text-lg">
+    Library
+  </a>
+</div>
 
-![](/cheerpj2/assets/cheerpj_visual_2.png)
+## What is CheerpJ?
 
 **CheerpJ is constituted of three components:**
 
```

Comment:
Let's simplify below point 1. Compiling single class file is theoretically possible but not supported by public facing tools, let's just remove that idea completely

Reply:
Also, about point 2) I am not sure what the intended meaning is. "Distributed in part or in full"

It would make more sense to mention the cloud vs self-hosting possibilities